README.md

```diff
@@ -47,6 +49,8 @@ You only need to run this command to build.
 
 > _Note:_ This is a convenient way to quickly build your firmware but using images from untrusted third-parties poses the risk of exposing your source code.
 
+> _Note 2:_ The prebuilt images are not available for `arm64` architecture (Apple M1), since GitHub Actions don't have hosted runners with Apple M1 yet.
```

Comment:
```suggestion
> _:green_apple: Note:_ The prebuilt images are not available for `arm64` architecture (Apple M1), because GitHub Actions don't have hosted runners with Apple M1 yet.
```